lab_1_keywords_tfidf/main.py

```diff
@@ -17,7 +17,22 @@ def clean_and_tokenize(text: str) -> Optional[list[str]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not isinstance(text, str):
+        return None
+
+    tokens = text.split()
```

Comment:
consider firstly removing punctuation and only then splitting into tokens, this way it'll be simpler